src/test/scala/com/gu/datalakealerts/integration/FullTestWithLocalPolling.scala

```diff
@@ -0,0 +1,41 @@
+package com.gu.datalakealerts.integration
+
+import com.gu.datalakealerts.EventModels.MonitoringEvent
+import com.gu.datalakealerts.{ Athena, Features, Platforms, SchedulerEnv, SchedulerLambda, WorkerLambda }
+import com.gu.datalakealerts.WorkerLambda.logger
+
+// Requires Ophan (Athena & Glue) and Dev Playground (Developer) Janus credentials
+object FullTestWithLocalPolling {
```

Comment:
Most developers who contribute to this project do not have `Ophan (Developer)` permissions. Many of them are client-side developers who don't have much experience with AWS/SQS. This functionality exists because I wanted to allow all developers to check that their alerts were set up correctly without needing lots of permissions / worrying about serverless complexity!

Reply:
I think I'll steal the idea of a Main class in the test namespace to run a lambda locally. In fact maybe I should change the lambda boilerplate to adopt that pattern